build.gradle

```diff
@@ -0,0 +1,274 @@
+
+
+    //  This property can be used to speed up an SDK build when no dynamic languages are needed
+    def excludeDynamicLanguages = findProperty('excludeDynamicLanguages')
+
+    // Register graal repository dummy file as an input and repository directory as output
+    inputs.files(graalCloneDummy)
+    outputs.files(graalBuildDummy)
+
+    doFirst {
+
+        assertDockerContainerIsBeingUsed()
+
+        workingDir "$graalDir/vm"
+
+        // The --force-bash-launchers makes the build faster and use far less RAM by skipping the native-imaging of
+        // some tools we don't use or care about. Without that flag the build can fail due to running out of RAM.
+        if (excludeDynamicLanguages != null) {
+        commandLine "${environment.MX_HOME}/mx", '--skip-libraries=polyglot', '--dynamicimports', '/substratevm',
+                '--exclude-components=nju,nic,llp,lg,gu,polynative,gvm,poly,nil,svml', '--force-bash-launchers=polybench', 'build'
+        } else {
+            commandLine "${System.getenv('MX_HOME')}/mx", '--skip-libraries=polyglot', '--dynamicimports', 'graalpython,graal-js,/substratevm',
+                   '--exclude-components=nju,nic,llp,lg,gu,polynative,gvm,poly,nil,svml', '--force-bash-launchers=python,js,polybench', 'build'
+        }
+    }
+
+    doLast {
+        // Write current date to graal build dummy file. If this task is run again, the content of this file
+        // will change and subsequent tasks will run.
+        writeDateToFile(graalBuildDummy)
+    }
+}
+
+task getGraalBuildFolder(type: Exec) {
+    dependsOn buildGraal
+
+    inputs.files(graalBuildDummy)
+    standardOutput = new ByteArrayOutputStream()
+
+    commandLine 'ls', "$graalVMLatestDir"
+    doLast {
+        ext.graalVMDir = graalVMLatestDir + standardOutput.toString().replaceAll('\\r', '').replaceAll('\\n', '')
+    }
+}
+
+/**
+ * The command line util `tar` is being used rather Gradle's Tar task due to
+ * https://github.com/gradle/gradle/issues/3982, which describes symlinks being followed
+ * rather than preserved. Graal fails to run with the resolved symlinks.
+ */
+task tarGraal(type: Exec) {
+    dependsOn getGraalBuildFolder
+
+    // Task details
+    group = 'Other'
+    description = 'Generate graalvm.tar file.'
+
+    // Set the input to the folder in which the build is generated. We actually want to tar the only subdirectory from this.
+    inputs.dir(graalVMLatestDir)
+    outputs.file(graalVMTar)
+
+    doFirst {
+        if (environment.JVMCI_HOME.contains('openjdk1.8')) {
+            commandLine 'tar', 'czf', "$graalVMTar", '-C', getGraalBuildFolder.graalVMDir,
+                    "--exclude=./bin",
+                    "--exclude=./lib",
+                    "--exclude=./man",
+                    "--exclude=./sample",
+                    "--exclude=*src.zip",
```

Comment:
We don't need to worry about excluding files now. The exclusions were added to reduce the size of the graalvm artifact in the sdk zip file, thus reducing the size of the final zip distribution. This is no longer an issue since graalvm will be downloaded via maven (and then cached). Plus some of these patterns are no longer valid it looks like.